pkg/utils/hex.go

```diff
@@ -0,0 +1,7 @@
+package utils
+
+import "encoding/hex"
+
+func HexEncode(data []byte) string {
+	return hex.EncodeToString(data)
+}
```

Comment:
Should we just move the `hex.go` from `/testutils/` here?

Reply:
Nice catch